### ISSUE TITLE
Remove zoom slider

### ISSUE THREAD

**Issue by sgratzl** [arclamp/materialnet#141]
![image](https://user-images.githubusercontent.com/4129778/66228822-6e419300-e6ae-11e9-9cef-57971eb8eb86.png)

do we need it, since everyone will probably just use the mouse anyhow?

**Comment by waxlamp** [comment]
Yes, I think we can get rid of this. I'd like to keep the "node spacing" slider there, however, since it's not as standard as the regular zoom.
